app/views/layouts/application.html.erb

```diff
@@ -74,10 +74,19 @@
     </div>
   </header>
 
-  <% flash.each do |name, message| %>
-    <div class='hackney-flash-message'>
+  <% flash.each do |name, messages| %>
+    <div class='hackney-flash-message hackney-flash-message-<%= name %>'>
       <div class="govuk-width-container">
-        <p><%= message %></p>
+        <% messages = [messages].flatten %>
+        <% if 1 == messages.count %>
+          <p><%= messages.first %></p>
+        <% elsif 1 < messages.count %>
+          <ul>
```

Comment:
How about keeping only the `ul` instead by adding a css class if the `count > 1` to render the bullets?

Reply:
I just didn't want to change the previous behaviour, but I agree this smells. Gonna fix :+1:

Reply:
done